### ISSUE TITLE
Light Crystals don't drop off of the low level enemies they're supposed to.

### ISSUE THREAD

**Issue by gm-bravura** [OmegaPrivateServer/issues#266]
## Summary
Light Crystals are 15k a stack because they don't drop off of the low level enemies they're supposed to. New players have zero access to this crystal. IG: (Level 1-5 Two of Coins)

## Relevant Wiki Link 
(IMPORTANT: AUGUST 2008 WIKI ENTRIES OR EARLIER)

## Characters
Thatswhat

## Steps to Reproduce

1.
2.
3.
4.

**Comment by Noukon** [comment]
Is that going to be fixed anyday? The same happens with Tonberries and Pots (or the drop rate is really lower than it should). Killed a bunch of those in the Temple of Uggalepih without seeing any Light crystals.

The current economy about these is only controlled by players with sky access, preventing lower level players from farming these crystals/clusters (GL having a reliable way of farming this in Qufim island, as Auroras appear very rarely), and have to depend on abusive prices on AH.

**Comment by zierk** [comment]
I'll have a look at this.

**Comment by zierk** [comment]
So good and bad news here. 

Good news: I have identified the reason for Cardians not dropping the correct crystal. I have submitted a patch fixing this. When the patch goes live Cardians "of Coins" will drop Light Crystals, "of Swords" = Fire Crystals, "of Batons" = Earth Crystals, "of Cups" = Water Crystals. 

Sorta-Bad news: Golems, Hippogryphs, Magic Pots, Mimics, Rocs, and Tonberries all drop light crystals. However, if they are in a region that is controlled by beastmen, no crystals will drop. Unfortunately this seems to be the likely case in most areas where light-yielding mobs are located.
